src/adagrams.js

```diff
@@ -1,15 +1,131 @@
 export const drawLetters = () => {
-  // Implement this method for wave 1
+    const LETTER_POOL = {
+      'A': 9, 
+      'B': 2, 
+      'C': 2, 
+      'D': 4, 
+      'E': 12, 
+      'F': 2, 
+      'G': 3, 
+      'H': 2, 
+      'I': 9, 
+      'J': 1, 
+      'K': 1, 
+      'L': 4, 
+      'M': 2, 
+      'N': 6, 
+      'O': 8, 
+      'P': 2, 
+      'Q': 1, 
+      'R': 6, 
+      'S': 4, 
+      'T': 6, 
+      'U': 4, 
+      'V': 2, 
+      'W': 2, 
+      'X': 1, 
+      'Y': 2, 
+      'Z': 1
+      };
+
+    const alphabet = 'ABCDEFGHIJKLMNOPQRSTUVWXYZ';
+
+    const hand = [];
+    let letterPool = JSON.parse(JSON.stringify(LETTER_POOL));
+    
+    while(hand.length < 10) {
+        const randomLetter = alphabet[Math.floor(Math.random() * alphabet.length)];
```

Comment:
You're grabbing a random letter from `alphabet` which only has 26 letters. Therefore, your chances of getting 'A' would be 1/26. We should randomly get a letter from a list that has 98 tiles (9 As, 2 Bs, etc), in this case you'd have a 9/98 chance of grabbing an A. 

While you do decrement the number of letters in the following `if` statement, you're still only getting a letter from 26 possible letters instead of 98.